src/test/js/multibranch/filtering.js

```diff
@@ -20,36 +20,25 @@ module.exports = {
                       .then(done);
               });
     },
-    
```

Comment:
I did not remove it but rather reworked the code a bit to be more stable as I found the test to be flaky and fail a few times. I broke one large step into three; for some reason, it seemed slightly more stable when broken up into steps, but that could just be a coincidence. I also added some more explicit "waitFor..." statements that made sure the DOM of the runs table had updated before asserting on the number of rows.